README.md

```diff
@@ -534,6 +534,18 @@ await $`pwd`
 EOF
 ```
 
+#### Attaching .bash_profile/.zshrc
+
+By default `child_process` does not include aliases and bash functions (https://github.com/nodejs/node/issues/3916). 
```

Comment:
I think link here unnecessarily as this behavior is actually expected. Not an "issue".

Reply:
ok